docs/guide/test-fixtures.md

```diff
@@ -333,6 +335,8 @@ yii fixture User --globalFixtures='some\name\space\Custom'
 Unloading fixtures
 ------------------
 
+This function does not clear the data at the moment. Because this function on discussing by developers.
```

Comment:
`unload()` method of fixtures should be triggered properly by this command. Could you provide steps to reproduce the issue?